### DISCUSSION TITLE
Marp PDF doesn't work with latest chrome (canary)

### DISCUSSION THREAD

**Discussion by Hultner** [marp-team/marp#259] [Q&A] [answered]
I ran into some problems today with generating PDF slide-decks from marp. Turns out that marp is incompatible with the most recent Chrome version. I added an issue about this in marp core but thought it would be an good idea to mention it here in case other have the same problem and panic in generating PDFs. What I did instead was to use an older chromium version with marp and that worked like a charm.

This can be set via `CHROME_PATH=/.../...` env var for CLI or via extension setting for vscode.

**Comment by yhatt** [comment, marked as answer]
**UPDATE**: It has been fixed by https://chromium-review.googlesource.com/c/chromium/src/+/3428144. Please confirm you are using Google Chrome Canary 100.0.4863.0 or later.

---

See details: https://bugs.chromium.org/p/chromium/issues/detail?id=1292341, marp-team/marp-cli#419

Marp team has verified this problem when using Chrome Canary 100.0.4859.0 for conversion to PDF, PPTX, and images. It seems to be an upstream bug of Chrome/Chromium.

Currently Chrome v99 and earlier versions have no problems about PDF conversion. At this time, the latest stable Chrome is v97 so the most of users should not meet to this bug. (I have made clear the subject of this thread that this problem is caused by canary)

**Comment by pdumin** [comment]
After generating the pdf, the first few slides become like this. Is this also related to the Chrome version?

`body,html{-ms-scroll-snap-points-y:repeat(100%);scroll-snap-points-y:repeat(100%);background:#000;height:100%;margin:0;overflow:auto;-ms-scroll-snap-type:y mandatory;scroll-snap-type:y mandatory;-ms-scroll-snap-type:mandatory;scroll-snap-type:mandatory}body>svg{display:block;height:100%;scroll-snap-align:center center;width:100%}svg>foreignObject>section{width:1280px;height:720px;box-sizing:border-box;overflow:hidden;position:relative;scroll-snap-align:center center}svg>foreignObject>section:after{bottom:0;content:attr(data-marpit-pagination);padding:inherit;pointer-events:none;position:absolute;right:0}svg>foreignObject>section:not([data-marpit- pagination]):after{disp`


UPD. Downgrading Chrome to 97.0.4692.99 solved the problem.

**Reply by yhatt** [reply]
Yes. It seems to be a bug of Chrome Canary, not Marp. Refer to Chromium bug tracker: https://bugs.chromium.org/p/chromium/issues/detail?id=1292341
